src/command.rs

```diff
@@ -54,9 +54,17 @@ impl Command {
       Self::Operation(operation) => state.operation = *operation,
       Self::Print => state.print()?,
       Self::Repl => {
-        for result in BufReader::new(io::stdin()).lines() {
-          let line = result?;
-          match line.trim().parse::<Self>() {
+        let history = tilde("~/.degenerate_history").to_string();
+
+        let mut rl = Editor::<()>::new();
+
+        if rl.load_history(&history).is_err() {
+          eprintln!("Created history file in: {}", history);
+        }
```

Comment:
I think it's okay to not print anything:

```suggestion
        rl.load_history(&history).ok();
```